packages/@dcl/react-ecs/src/components/uiTransform/utils.ts

```diff
@@ -52,6 +52,10 @@ function parsePositionUnit(val?: PositionUnit): [number | undefined, YGUnit] {
     return [undefined, YGUnit.YGU_UNDEFINED]
   }
 
+  if (val === 'auto') {
+    return [Number(val), YGUnit.YGU_AUTO]
```

Comment:
Number('auto') will return NaN 🤔 .
 Maybe this needs to return [undefined, YGUint.YGU_AUTO] ?